qiita_pet/templates/sitebase.html

```diff
@@ -133,6 +133,14 @@
                 <li><a href="/study/list/">View Studies</a></li>
               </ul>
             </li>
+            <li class="dropdown">
+              <a href="/user/messages/">Messages</a>
+              <ul class="dropdown-menu">
+              {% for msg, timestamp in current_user.messages(5) %}
```

Comment:
This can be easily accomplish with a UI module in which you insert that staff in the bar, so we can remove any database access in the template.

This should be applied for the entire navigation bar, but that is out of scope of the PR, but I think we should not be adding more potential issues to the code.